app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -52,6 +89,14 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
         fun isURLPinned(): Boolean
     }
 
+    private lateinit var onUrlEnteredListener: OnUrlEnteredListener // default impl does nothing.
+
+    /**
```

Comment:
\[detekt\] INFO: \[comments/CommentOverPrivateProperty\] Private properties should be named such that they explain themselves even without a comment\.